pkgs/development/libraries/openal-soft/default.nix

```diff
@@ -23,6 +23,11 @@ stdenv.mkDerivation rec {
   postPatch = ''
     substituteInPlace core/helpers.cpp \
       --replace "@OUT@" $out
+    # https://github.com/kcat/openal-soft/pull/696
```

Comment:
Can we fetchpatch this?

Reply:
Sure, let me do that. For some reason I thought I was going with something different from the PR, but nope, I was probably confused.